src/ResourceManagement/Compute/Microsoft.Azure.Management.Compute/Generated/VirtualMachinesOperations.cs

```diff
@@ -2393,165 +2402,6 @@ internal VirtualMachinesOperations(ComputeManagementClient client)
         }
 
         /// <summary>
-        /// The operation to list virtual machines under a resource group.
-        /// </summary>
-        /// <param name='nextPageLink'>
-        /// The NextLink from the previous successful call to List operation.
-        /// </param>
-        /// <param name='customHeaders'>
-        /// Headers that will be added to request.
-        /// </param>
-        /// <param name='cancellationToken'>
-        /// The cancellation token.
-        /// </param>
-        /// <return>
-        /// A response object containing the response body and response headers.
-        /// </return>
-        public async Task<AzureOperationResponse<IPage<VirtualMachine>>> ListNextWithHttpMessagesAsync(string nextPageLink, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
@amarzavery 
Confirmed by the service API owners, these APIs don't have pagination, so that it's not the cause that the customers are using them, thus they won't face this as a breaking change.